### PR TITLE
[WIP][integration] Performance tests using Locust and taurus

### PR DESCRIPTION
**Pull request by 97amarnathk**
BEGINRELEASENOTES

Please follow the template:
* ResourceStatusSystem
NEW: Performance test for ResourceStatusSystem using Locust and Taurus

ENDRELEASENOTES

### CODE REVIEW

**Comment by atsareg** [comment]
These tests include installation of the locustio package. pip install says that this package is only valid for python >= 3.6,  I guess this is the reason for the test failure. So, these tests can only be made when we will migrate to Python 3. Am I missing something ?

**Comment by 97amarnathk** [comment]
I think Locust framework is available for Python 2.7 as well.

https://pypi.org/project/locustio/

**Comment by fstagni** [comment]
It's maybe enough to rebase to the current `integration` branch.

**Comment by atsareg** [comment]
What I see is:

    # pip install locustio
    Collecting locustio
      Using cached https://files.pythonhosted.org/packages/bd/50/c4bafccbb1dd66a1968f3f6d96933889b338475f4a8e45a7785f7f275fc5/locustio-0.999.tar.gz
    ERROR: Package 'locustio' requires a different Python: 2.7.13 not in '>= 3.6'

So, apparently there are problems with the locustio for Python 2

**Comment by chrisburr** [comment]
The locustio metadata on PyPI isn't configured correctly but this works for getting the last version with support for Python 2.7:

```bash
pip install 'locustio<0.14'
```

**Comment by atsareg** [comment]
Can this be expressed properly in the requirements.txt ?

File: requirements.txt [hunk 1]
```diff
@@ -16,6 +17,7 @@ funcsigs
 future
 futures
 GitPython>=2.1.0
+locustio
```
Comment (andresailer) [inline requirements.txt:20]:
```suggestion
locustio<0.14
```

Comment (andresailer) [reply requirements.txt:20]: Actually also the environment.yml file for the CI, I guess.

**Pull request closed without merging**